stglib/exo.py

```diff
@@ -526,16 +532,33 @@ def add_attributes(var, dsattrs):
             }
         )
 
-    for var in ds.variables:
-        if (var not in ds.coords) and ("time" not in var):
-            add_attributes(ds[var], ds.attrs)
+    # for var in ds.variables:
+    #    if (var not in ds.coords) and ("time" not in var):
+    #        add_attributes(ds[var], ds.attrs)
 
     return ds
 
 
 def read_exo_header(filnam, encoding="utf-8"):
     header = {}
     try:
+        # new version of KOR export file
```

Comment:
Are you sure that reversing the order in the try/except blocks will still work successfully on both old and new versions?